### ISSUE TITLE
Extremely high sample rate for cries

### ISSUE THREAD

**Issue by ExpoSeed** [rh-hideout/pokeemerald-expansion#1332]
Currently, cries in Pokemon Expansion are sampled at a rate of 44.1 kHz. This is extremely high for the GBA, and as a result, Pokemon Expansion takes around 29 MB.

UltimaSoul: "The problem is that they're exported at a higher sample rate as .aif files which is generally higher than the ripped versions so it takes more space unecessarily. Secondly, they're currently uncompressed, so likely it'd be a smarter move to not used uncompressed cries."

**Comment by MissingNoL** [comment]
^ Unsure what that commit does, but I am wondering if this is an issue still?  I bet you can use Audacity's "export multiple" option to change these all in one move.  I think you can change the sample rate for all files in addition to using a compressed format: https://manual.audacityteam.org/man/exporting_audio.html
If so, what format and sample rate is needed?

**Comment by ultima-soul** [comment]
Gen 1-5 Pokemon are resolved for this issue thanks to Lunos and I. Gen 6-8 and relevant forms are a WIP.

**Comment by LOuroboros** [comment]
I'm using this Issue ticket as a discussion point.

I think merging #2010 is a no go because the cries from USUM all take more space compared to what the PE currently uses. There's a majority that is opposed to merging it in as a result *(me included)*. We should be saving as much space as possible, not the opposite.

Now, staying true to the concerns of this ticket, I downsampled every cry in the PE to 10512hz using ffmpeg, the Windows Command Prompt and a nice `for` loop *(`FOR /F "tokens=*" %G IN ('dir /b *.aif') DO ffmpeg -y -i "%G" -acodec pcm_s8 -ar 10512 "new\%~nG.aif"`)*, and I think there's a sustantial gain in space.

A PE ROM built without padding using those downsampled cries weighs 21.2 MB.

Naturally, this comes at a cost; audio quality.

I can't attach the video directly because GitHub has a 10mb file size limit, but for those who're inside the Discord server:

https://discord.com/channels/419213663107416084/774393519569502268/927240977868025886

**Comment by LOuroboros** [comment]
So, I tried a different experiment after ZChaotix posed a good question in the server.

Turns out that using the default PE cries, changing every instance of `cry_uncomp` and `cry_reverse_uncomp` in `cry_tables.inc` to `cry` and `cry_reverse` respectively and then removing the `uncomp_` from the cries of each species from Chespin to Calyrex, a PE ROM without padding weighs 21.8 MB.

I think this could be the best way to go personally.

Even Dwebble's cry sounds pretty decent.

https://user-images.githubusercontent.com/4485172/147884142-1c0c7a0e-e4e1-449a-ad88-ac06c20c3e18.mp4

https://user-images.githubusercontent.com/4485172/147884155-16ac9862-3eb1-4917-9828-c8f3b6d77017.mp4
